### PR TITLE
feat: /azure redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied changes as per [doc](https://docs.google.com/document/d/1GUlSATrqZmGrsk4DhP9W0s-gcOuFxpdKVgx7YE4g780/edit) and [design](https://www.figma.com/design/MinLlXouBnEDqaoAL3bTWC/24.10-ubuntu.com%2Fazure?node-id=251-1946&node-type=canvas&t=c1gK4qgVqbBBn9vl-0)

## QA

- View the site locally in your web browser at: https://ubuntu-com-14292.demos.haus/azure
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12821

### CODE REVIEW

**Comment by juanruitina** [comment]
Just some tiny inconsistencies:

- [ ] Replace "24x7" with "24/7"
- [ ] Add chevrons to the links under "Get Ubuntu Pro on Azure now"
- [ ] "Talk to us about Ubuntu on Azure" in the footer is lacking some space between HR and button

+1ing this already as these are minor.

**Comment by lyubomir-popov** [comment]
looks great, a few tiny niggles:

- p-section--shallow missing from the hero (both headings, and paragraph should be in them). See the example [here](image)

image

- this should be shallow:
 
image

- line touching button:

image

- last section should always be deep:
image

**Comment by mtruj013** [comment]
@lyubomir-popov @juanruitina thanks both!

> "Talk to us about Ubuntu on Azure" in the footer is lacking some space between HR and button

> line touching button:

didn't address these as they'll be fixed once we update to the newest Vanilla (which will be done before this is released)

File: templates/azure/index.html [hunk 1]
```diff
@@ -25,61 +31,66 @@ <h1>Ubuntu on Azure</h1>
...
-      <div class="col-12">
-        <h2 class="p-muted-heading">Ubuntu on Azure powers cloud leaders like</h2>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule--muted" />
+      <div class="col p-section--shallow">
+        <h2>
+          Ubuntu on Azure powers
+          <br />
+          cloud leaders
```
Comment (akbarkz) [inline templates/azure/index.html:60]: in the copy doc it says "...leaders like"

File: templates/azure/index.html [hunk 2]
```diff
@@ -25,61 +31,66 @@ <h1>Ubuntu on Azure</h1>
           <p>
             Beyond Ubuntu Server images, Canonical offers a number of premium Ubuntu images that extend Ubuntu Server with additional security and management features, billed through your existing Microsoft Azure account.
           </p>
-          <a href="/azure/contact-us" class="p-button--positive js-invoke-modal">Talk to us about Ubuntu on Azure</a>
         </div>
-        <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
+        <hr class="is-muted" />
```
Comment (akbarkz) [inline templates/azure/index.html:35]: shall we fix this to `p-rule--muted`?

File: templates/azure/index.html (hunk 1 above)
Comment (mtruj013) [reply templates/azure/index.html:60]: Double checked with Juan and the doc was just out of date, the "like" was supposed to be dropped. He's updated the doc

File: templates/azure/index.html [hunk 3]
```diff
@@ -88,40 +99,59 @@ <h2 class="p-muted-heading">Ubuntu on Azure powers cloud leaders like</h2>
...
+          </div>
+          <hr class="p-rule--muted" />
+          <ul class="p-list--divided p-section--shallow">
             <li class="p-list__item is-ticked">Azure-optimised kernel and userspace components built from the latest releases</li>
             <li class="p-list__item is-ticked">Broad set of open source application components available for development</li>
             <li class="p-list__item is-ticked">Security patches for the kernel and key infrastructure components</li>
             <li class="p-list__item is-ticked">Updates mirrored and images published in all Azure regions worldwide</li>
             <li class="p-list__item is-ticked">5-year lifetime</li>
           </ul>
+          <hr class="is-muted" />
```
Comment (akbarkz) [inline templates/azure/index.html:135]: Please replace the classname here too

File: templates/azure/index.html [hunk 4]
```diff
@@ -131,120 +161,114 @@ <h3 class="p-card__title">Ubuntu Pro for Azure</h3>
...
+              FIPS-certified modules and CIS audit/hardening tooling for Ubuntu Pro 22.04 are in progress, launch <a href="https://azuremarketplace.microsoft.com/en-gb/marketplace/apps/canonical.0001-com-ubuntu-pro-jammy?tab=Overview">Ubuntu Pro</a> and activate the <a href="/blog/ubuntu-22-04-fips-140-3-modules-available-for-preview">fips-preview channel</a> for a preview, or use <a href="https://azuremarketplace.microsoft.com/en-gb/marketplace/apps/canonical.0001-com-ubuntu-pro-focal-fips?tab=Overview">Ubuntu Pro FIPS 20.04 LTS</a>
             </li>
             <li class="p-list__item is-ticked">
-              Optional 24/7 support for your Ubuntu workloads (<a href="/azure#get-in-touch">contact us</a> to discuss adding this by Azure Marketplace Private Offer)
+              Optional 24/7 support for your Ubuntu workloads (<a href="/azure/contact-us">contact us</a> to discuss adding this by Azure Marketplace Private Offer)
+            </li>
+            <li class="p-list__item is-ticked">10-year lifetime</li>
+          </ul>
+          <hr class="is-muted" />
```
Comment (akbarkz) [inline templates/azure/index.html:172]: and here

**Review by akbarkz — APPROVED** [review]

**Pull request merged**